builtin/providers/azurerm/resource_arm_template_deployment.go

```diff
@@ -155,20 +155,44 @@ func resourceArmTemplateDeploymentRead(d *schema.ResourceData, meta interface{})
 	if resp.Properties.Outputs != nil && len(*resp.Properties.Outputs) > 0 {
 		outputs = make(map[string]string)
 		for key, output := range *resp.Properties.Outputs {
+			log.Printf("[DEBUG] Processing deployment output %s", key)
 			outputMap := output.(map[string]interface{})
 			outputValue, ok := outputMap["value"]
 			if !ok {
 				// No value
 				continue
 			}
+			outputType := outputMap["type"]
+			var outputStringValue string
+
+			switch outputType {
+			case "Bool":
+				if outputValue == false {
+					outputStringValue = "0"
+				} else if outputValue == true {
+					outputStringValue = "1"
+				} else {
+					return fmt.Errorf("Invalid value %s for boolean output %s", outputValue, key)
+				}
```

Comment:
Regarding Booleans, the documentation in https://www.terraform.io/docs/configuration/variables.html is a little confusing, but it looked to me like I should be explicitly using "0" and "1" instead of "false" and "true".  Is that correct?  If I set an output in a .tf file to be based off of a Boolean variable, I get "0" and "1" values.  (Also, as far as I can tell, there's no way to configure azurerm_template_deployment.outputs to be a list of interface{}.)  Either way, it should be "false"/"true", not "False/True", right?